source/kraken/worker.cpp

```diff
@@ -780,6 +786,19 @@ void Worker::pt_ref(const pbnavitia::PTRefRequest& request) {
         boost::make_optional(request.has_until_datetime(), bt::from_time_t(request.until_datetime())), rt_level, *data);
 }
 
+// returns true if there is an error
+bool Worker::set_isochrone_distributed_args(const pbnavitia::JourneysRequest& request,
+                                            JourneysArg& arg,
+                                            const std::string& name) {
+    auto has_error = set_journeys_args(request, arg, name);
+    if (!has_error && !arg.isochrone_center) {
+        err_msg_isochron(this->pb_creator, "isochrone_distributed needs an isochrone_center");
```

Comment:
The check that an `isochrone_center` must exist can be done much earlier.
It's a pre-requisite to `isochrone_distributed`.

```cpp
void Worker::isochrone_distributed(const pbnavitia::JourneysRequest& request){
  if (!request.has_isochrone_center()) {
        err_msg_isochron(this->pb_creator, "isochrone_distributed needs an isochrone_center");
  [...]
```

By doing the check earlier, this function can be removed.

Reply:
Done! `set_isochrone_distributed_args` is no more :cry: